### ISSUE TITLE
Create Wasserstein metric for ABC discrepancy measures

### ISSUE THREAD

**Issue by ben18785** [pints-team/pints#888]

**Comment by MichaelClerx** [comment]
Are you planning to sample some metric that compares 2 probability distributions?

**Comment by ben18785** [comment]
The idea is to compare the raw data themselves through this metric. I should have put a reference to the paper I was thinking of before. It's [this](https://arxiv.org/pdf/1905.03747.pdf) one.

**Comment by MichaelClerx** [comment]
Cool! Are they fitting rather than sampling with ABC here? It sounds suspiciously like they want to fit a distribution to me:

> propose to avoid the use of summaries and the ensuing loss of information by instead using the Wasserstein distance between the empirical distributions of the observed and synthetic data.

**Comment by ben18785** [comment]
Nope. They are comparing the empirical distribution of the actual and simulated data!
